api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -350,6 +350,38 @@ public void getDateFormat_shouldReturnAPatternWithFourYCharactersInIt() {
 		Assert.assertEquals("tt.MM.uuuu", OpenmrsUtil.getDateFormat(Locale.GERMAN).toLocalizedPattern());
 		Assert.assertEquals("dd-MM-yyyy", OpenmrsUtil.getDateFormat(new Locale("pt", "pt")).toLocalizedPattern());
 	}
+	@Test
+	public void getDateFormat_shouldReturnTrueWhenDateIsNotNull() {
+		Date date = null;
+		try {
```

Comment:
why you do try and catch here

Reply:
Since am testing a method of "getDateFormat" and the corresponding method , i added dummy data to bring it the right way.  Thought Adding try and catch would bring the aspect of asserting the statements